server/routes/goalRoutes.ts

```diff
@@ -22,7 +22,7 @@ router.get('/get/module/:id', tokenMethods.authenticateToken, async(req : any, r
 
 router.post('/add', tokenMethods.authenticateToken, async(req : any, res : any) => {
```

Comment:
Now that you're using typescript, it might be worth switching everything over to using modules so you can import them like this `import express, { Request, Response } from 'express';`. This will also let you grab the types for your routes so you can get rid of the `any` type you have there

Reply:
Well, I believe I tried this but when I ran my unit tests it still wasn't able to recognize the es6 module imports, so I just stuck with using common js modules for now.